dotnetv3/DynamoDB/scenarios/PartiQL_Basics_Scenario/PartiQL_Basics_Scenario/PartiQLMethods.cs

```diff
@@ -0,0 +1,244 @@
+
+            return response.HttpStatusCode == System.Net.HttpStatusCode.OK;
+        }
+
+        // snippet-end:[PartiQL.dotnetv3.PartiQLBasicsScenario-InsertSingleMovie]
+
+        // snippet-start:[PartiQL.dotnetv3.PartiQLBasicsScenario-UpdateSingleMovie]
+
+        /// <summary>
+        /// Updates a single movie in the table, adding information for the
+        /// Producer.
+        /// </summary>
+        /// <param name="tableName">the name of the table.</param>
+        /// <param name="producer">The name of the producer.</param>
+        /// <param name="movieTitle">The movie title.</param>
+        /// <param name="year">The year the movie was released.</param>
+        /// <returns>A Boolean value representing th success or failure of the
+        /// UPDATE operation.</returns>
+        public static async Task<bool> UpdateSingleMovie(string tableName, string producer, string movieTitle, int year)
+        {
+            string insertSingle = $"UPDATE {tableName} SET Producer=? WHERE title = ? AND year = ?";
+
+            var response = await Client.ExecuteStatementAsync(new ExecuteStatementRequest
+            {
+                Statement = insertSingle,
+                Parameters = new List<AttributeValue>
+                {
+                    new AttributeValue { S = producer },
+                    new AttributeValue { S = movieTitle },
+                    new AttributeValue { N = year.ToString() },
+                },
+            });
+
+            return response.HttpStatusCode == System.Net.HttpStatusCode.OK;
+        }
+
+        // snippet-end:[PartiQL.dotnetv3.PartiQLBasicsScenario-UpdateSingleMovie]
+
+        // snippet-start:[PartiQL.dotnetv3.PartiQLBasicsScenario-DeleteSingleMovie]
+
+        /// <summary>
+        /// Delete a single move from the table.
+        /// </summary>
+        /// <param name="tableName">The name of the table.</param>
+        /// <param name="movieTitle">The title of the movie to delete.</param>
+        /// <param name="year">The year that the move was released.</param>
+        /// <returns>A Boolean value indicating the success of failure of the
+        /// DELETE operation.</returns>
+        public static async Task<bool> DeleteSingleMovie(string tableName, string movieTitle, int year)
+        {
+            var deleteSingle = $"DELETE FROM {tableName} WHERE title = ? AND year = ?";
+
+            var response = await Client.ExecuteStatementAsync(new ExecuteStatementRequest
+            {
+                Statement = deleteSingle,
+                Parameters = new List<AttributeValue>
+                {
+                    new AttributeValue { S = movieTitle },
+                    new AttributeValue { N = year.ToString() },
+                },
+            });
+
+            return response.HttpStatusCode == System.Net.HttpStatusCode.OK;
+        }
+
+        // snippet-end:[PartiQL.dotnetv3.PartiQLBasicsScenario-DeleteSingleMovie]
+
+        /// <summary>
+        /// 
```

Comment:
Is the summary missing here?

Reply:
Yes, it was. Updated with the proper information.